ui/hooks/useAccountTrackerPolling.ts

```diff
@@ -0,0 +1,25 @@
+import { useSelector } from 'react-redux';
+import { getNetworkConfigurationsByChainId } from '../selectors';
+import {
+  accountTrackerStartPolling,
+  accountTrackerStopPollingByPollingToken,
+} from '../store/actions';
+import useMultiPolling from './useMultiPolling';
+
+const useAccountTrackerPolling = () => {
+  // Selectors to determine polling input
+  const networkConfigurations = useSelector(getNetworkConfigurationsByChainId);
+  const availableNetworkClientIds = Object.values(
+    networkConfigurations,
+  ).flatMap((networkConfiguration) =>
+    networkConfiguration.rpcEndpoints.map(
+      (rpcEndpoint) => rpcEndpoint.networkClientId,
+    ),
+  );
```

Comment:
We don't need to poll across every rpc endpoint, but rather just the default endpoint for each chain.  So I think this would be something like

```suggestion
  const availableNetworkClientIds = Object.values(networkConfigurations).map(
    (networkConfiguration) =>
      networkConfiguration.rpcEndpoints[
        networkConfiguration.defaultRpcEndpointIndex
      ].networkClientId,
  );
```

Reply:
Sorry am i wrong or both suggestions have the same result?

Reply:
If there are multiple rpc endpoints configured for a network, it looks like the old code would poll all of them.  Whereas the suggested code would only poll the default endpoint.